srcpkgs/fontconfig/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'fontconfig'
 pkgname=fontconfig
 version=2.13.1
-revision=2
+revision=3
```

Comment:
Do we want to revbump this or just make the change and wait for it to happen through attrition?

Reply:
Same question for mkfontscale

Reply:
Why not they're both small packages. Also we've been dropping noarch through revbumps for quite some time now...

https://github.com/void-linux/void-packages/pull/25929/commits/84af860c6256d2d6e7b3c63080abd8122d86c5f3 and similar in that PR
https://github.com/void-linux/void-packages/pull/25710
https://github.com/void-linux/void-packages/pull/25413

If I am understanding your question correctly...

Reply:
@abenson Yes, goal is to get rid of noarch packages from binary repos, and to do that in finite time revbumping them all is required.